lib/Dialect/ESI/ESIPasses.cpp

```diff
@@ -245,19 +300,242 @@ LogicalResult PipelineStageLowering::matchAndRewrite(
   return success();
 }
 
+namespace {
+/// Convert all the ESI ports on modules to some lower construct. SV interfaces
+/// for now. In the future, it may be possible to select a different format.
+struct ESIPortsPass : public LowerESIPortsBase<ESIPortsPass> {
+  void runOnOperation();
+
+private:
+  void updateFunc(RTLExternModuleOp mod);
+  void updateInstance(RTLExternModuleOp mod, InstanceOp inst);
+  ESIRTLBuilder *B;
```

Comment:
nit: a single letter, capital, instance variable isn't the most clear. maybe spell out `builder` or something?